src/services/ledger/schema.ts

```diff
@@ -110,6 +111,10 @@ const transactionSchema = new Schema({
     type: Number,
     min: 0,
   },
+  _payment: {
```

Comment:
Naming this as `_payment` to be consistent with other places where we cross-reference collections (`_journal`, `_transactions`) by their mongoDB ObjectId